tests/chiltepin.yaml

```diff
@@ -4,6 +4,6 @@ provider:
   account: ""
 
 environment:
-  - "export FLUX_PMI_LIBRARY_PATH=/opt/admin/miniconda3/envs/chiltepin/lib/flux/libpmi.so"
+  - "export FLUX_PMI_LIBRARY_PATH=/opt/views/._view/lfvnahtxguhhjzbyvbk3ozvij43h244n/lib/flux/libpmi.so"
```

Comment:
Where does this arbitrary text, `lfvnahtxguhhjzbyvbk3ozvij43h244n` come from?

Reply:
That is the installation path of flux in the container.  When Spack installs packages, it uses a huge random string to create unique paths.  I know it's not ideal, but for now I don't have a better way to handle it.  Later we might consider trying to find a way to derive that path automatically at runtime.